bentoml/utils/log.py

```diff
@@ -97,11 +95,13 @@ def get_logging_config_dict(logging_level, base_log_directory):
             "dev": {"format": DEV_LOG_FORMAT},
             "prediction": {
                 "()": "pythonjsonlogger.jsonlogger.JsonFormatter",
-                "fmt": PREDICTION_LOG_JSON_FORMAT,
+                "fmt": "%%(service_name)s %%(service_version)s %%(api)s "
```

Comment:
@ssheng @yubozhao do we need to update related docs?

Reply:
We do. Good call. Will do in a separate PR.

Reply:
Fixed in https://github.com/bentoml/BentoML/pull/1488.